lib/carrierwave_data_uri/mount.rb

```diff
@@ -8,9 +8,12 @@ def mount_uploader(column, uploader=nil, options={}, &block)
         super
 
         class_eval <<-RUBY, __FILE__, __LINE__+1
+          attr_accessor :#{column}_data_uri
           attr_accessor :#{column}_data_filename, :#{column}_data_mimetype
 
           def #{column}_data_uri=(data)
+            return if data.blank?
```

Comment:
As I said before this can break existing integrations of this gem, so we need to remove it from here.
Also, I just pushed one commit to master where I cover this case.
So please rebase/merge the latest master into your branch.